### PR TITLE
status: 2023q1: pkgbase.live: suggested changes

### PR DESCRIPTION
**Pull request by grahamperrin**
@igalic

### CODE REVIEW

File: website/content/en/status/report-2023-01-2023-03/pkgbase.live.adoc [hunk 1]
```diff
@@ -7,18 +7,18 @@ link:https://codeberg.org/pkgbase/website[Website source] URL: link:https://code
...
-The website, and with it the link:https://codeberg.org/pkgbase/website/src/branch/main/howto/howdo.md[How Did She Do it?!] is still available in link:https://codeberg.org/pkgbase/website[Git].
+The website, and with it the link:https://codeberg.org/pkgbase/website/src/branch/main/howto/howdo.md[How Did She Do it?!] are still available in link:https://codeberg.org/pkgbase/website[Git].
 I highly encourage copy-cats.
 
-But I will also happily accept a new hardware sponsor!
+I will also happily accept a new hardware sponsor!
 
-Please note that I _have_ contacted the FreeBSD project, and they _are_ working on integrating PkgBase into release engineering.
-However, they are not yet ready, and they also cannot "simply" take over PkgBase.live because it uses a completely different process.
+Please note that I _have_ contacted the FreeBSD Project, and they _are_ working on integrating PkgBase into release engineering.
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-01-2023-03/pkgbase.live.adoc:23]: I approve this change specifically.

**Review by lsalvadore — COMMENTED** [review]
I will trust @igalic on approving this or not. Except for the correction of `project` into `Project` and maybe the edit of the link macro, I would have left everything as it is.

**Comment by grahamperrin** [comment]
Thanks, 

> the link macro

For clarity: the macro aspect is not the issue. 

The linked site describes itself: 

* as up.bsd._lv_
* not as (live) up.bsd._live_.

**Comment by lsalvadore** [comment]
> Thanks,
> 
> > the link macro
> 
> For clarity: the macro aspect is not the issue.
> 
> The linked site describes itself:
> 
> * as up.bsd._lv_
> * not as (live) up.bsd._live_.

Thanks @grahamperrin, I had not noticed it. I definitely approve this as well then: it is a necessary change to avoid confusion.

**Review by igalic — APPROVED** [review]
👍

**Comment by grahamperrin** [comment]
@igalic thank you, and I do hope to see PkgBase.live revitalised. 

----

Incidentally, you might like to update <https://github.com/igalic#projects-i-most-actively-work-on-%EF%B8%8F> in due course; maybe with reference to this quarter's report, when it appears.

**Pull request closed without merging**